src/uwtools/rocoto.py

```diff
@@ -21,42 +19,43 @@
 
 
 def realize_rocoto_xml(
-    config_file: OptionalPath,
-    output_file: OptionalPath = None,
-) -> bool:
+    config: Union[OptionalPath, YAMLConfig], output_file: OptionalPath = None
```

Comment:
Before our latest discussion in Slack about which in-memory support we really wanted to add at the moment, I'd already added support for realizing a Rocoto XML file from an in-memory `YAMLConfig` object in addition to a file support. I've since removed that option from the API but am hesitant to remove it from the library, in case we change our minds about this. So, it's not exposed to users, but it's latent (and unit-tested) capability in case we need it later. I can remove if there are objections.